Client.xcodeproj/xcshareddata/xcschemes/Fennec_Enterprise_XCUITests.xcscheme

```diff
@@ -168,8 +168,8 @@
    </BuildAction>
    <TestAction
       buildConfiguration = "Fennec"
-      selectedDebuggerIdentifier = ""
-      selectedLauncherIdentifier = "Xcode.IDEFoundation.Launcher.PosixSpawn"
+      selectedDebuggerIdentifier = "Xcode.DebuggerFoundation.Debugger.LLDB"
```

Comment:
@SimonBasca I know we no longer have the debugger by setting the identifier to `PosixSpawn`. Could we now run multiple tests at once on XCode IDE while having access to the debugger?

Reply:
@SimonBasca I tried this configuration (`Xcode.DebuggerFoundation.Debugger.LLDB`) and got the black screen after running more than one test. 🙇🏼 Please use the existing configuration (`Xcode.IDEFoundation.Launcher.PosixSpawn`) for `Fennec`. I have had discussion with the iOS developers why this configuration has been used.